qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -269,12 +303,12 @@ def __pow__(self, power):
         raise NotImplementedError("AddIntoPhaseGrad.__pow__ defined only for powers +1/-1.")
 
     def add_my_tensors(
-        self,
-        tn: 'qtn.TensorNetwork',
-        tag: Any,
-        *,
-        incoming: Dict[str, 'SoquetT'],
-        outgoing: Dict[str, 'SoquetT'],
+            self,
```

Comment:
`check/format-incremental --apply` from the repository root will apply the automated formatter